components/picture_selector.tsx

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState, useEffect} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {localizeMessage} from 'utils/utils';
+import * as FileUtils from 'utils/file_utils';
+
+import './picture_selector.scss';
+
+type Props = {
+    src?: string;
+    defaultSrc?: string;
+    loadingPicture?: boolean;
+    onSelect: (file: File) => void;
+    onRemove: () => void;
+};
+
+const PictureSelector: React.FC<Props> = (props: Props) => {
```

Comment:
1/5 - it would be cleaner do descructure the props: `({src,defaultSrc,loadingPicture,onSelect,onRemove}:Props)`
this will clean up the function a bit

Reply:
I like the cleaner look of the declaration personally, but I will destructure at the top of the function in a `const` so that we get basically the same effect.